src/app/Events/EventTemplates.tsx

```diff
@@ -115,8 +115,10 @@ export const EventTemplates = () => {
 
   React.useEffect(() => {
     addSubscription(
-      context.target.target().subscribe(refreshTemplates)
-    );
+      context.target.target().subscribe(() => {
+        setFilterText(filterText => filterText = '');
```

Comment:
Would simply `setFilterText('')` also work here?

Reply:
It does, thanks